DisguiseUnityRenderStream/Editor/DisguiseRenderStreamBuild.cs

```diff
@@ -172,5 +177,54 @@ static void AddSceneToSchema(ManagedSchema schema, int sceneIndex, string name)
             
             currentScene.parameters = parameters.ToArray();
         }
+        
+        static void CheckVsync()
+        {
+            if (DisguiseFramerateManager.Enabled && DisguiseFramerateManager.VSyncIsEnabled)
+            {
+                Debug.LogWarning($"DisguiseRenderStream: {nameof(QualitySettings)}{nameof(QualitySettings.vSyncCount)} is currently enabled. For best performance disable vSync in the project settings.");
```

Comment:
Minor: There is a missing dot between the two {...}.{...} sections: `{nameof(QualitySettings)}.{nameof(QualitySettings.vSyncCount)}`